src/rest.cpp

```diff
@@ -414,7 +413,7 @@ static bool rest_filter_header(const std::any& context, HTTPRequest* req, const
 
     switch (rf) {
     case RetFormat::BINARY: {
-        CDataStream ssHeader(SER_NETWORK, PROTOCOL_VERSION | RPCSerializationFlags());
+        CDataStream ssHeader(SER_NETWORK, PROTOCOL_VERSION);
```

Comment:
```suggestion
        CDataStream ssHeader{SER_NETWORK, PROTOCOL_VERSION};
```

Second commit: nit:

While touching this line, could use C++11 init?